### ISSUE TITLE
missing exec:Aviary.show 

### ISSUE THREAD

**Issue by andrea-86** [m1is/AviaryCordovaPlugin#18]
I create a new apache/cordova project with VisualStudio15 adn add the plugin from the git repo.
When i'm go to test the app with ripple i got this error message on console:

missing exec:Aviary.show 

TypeError: Cannot read property 'show' of undefined(…)
TypeError: Cannot read property 'show' of undefined
    at Object.module.exports.exec (ripple.js:40)
    at _show (Aviary.js:44)
    at Object.Aviary.show (Aviary.js:73)
    at onPhotoFileSuccess (index.js:47)
    at ripple.js:40
    at ripple.js:38

It seem to fail find native function of Aviary.

I also tryed to create the project from cordova cmd, adding plugin and building it for android and install the .apk file in my phone. It doesn't work and force the closing of the app after capturing the image.

In addition i see that when i build the project, in the androidManifest.xml my APIkey been always rewrite with the apiKey you use in you example.

What can be the problem?

Thanks

**Comment by kaansoral** [comment]
This plugin is very very old, since this plugin, adobe bought aviary, they also own phonegap, yet they don't develop a plugin themselves, bizarre

**Comment by m1is** [comment]
Yes it is weird that Adobe has not yet made an official Cordova plugin for Aviary, I think that's what everyone is banking on.  The problem with this particular plugin is that it was made for a particular version of the Aviary SDK (3.0 for Android, 3.3.11 for iOS), a particular version of Cordova (3.0) and particular versions of iOS and Android which makes this a difficult plugin to maintain.

**Comment by m1is** [comment]
Also this last December I received the following from Adobe in an email, looks like they are rebranding it Adobe Image Editor.  I don't know if this means that they may be ready to move forward with an official plugin but I am keeping my fingers crossed.

Deprecation of aviary.com - IMPORTANT
As you may know, Aviary joined the Adobe family over a year ago. The Adobe Creative Cloud provides the world’s best creative tools and content and now, with the Creative SDK, Adobe is committed to bringing that technology to developers. If you’d like to continue receiving these updates, pleasure make sure you opt-in for marketing emails during your initial Adobe ID sign up by checking the box labeled “Stay informed about Adobe products and services.”

We will be deprecating Aviary.com by the end of the year. The Aviary documentation will still be accessible but we will no longer be supporting Aviary API keys or accounts. Existing apps with Aviary API Keys will continue to work, however, you will need a new API Key when you transition to the most recent version of the Image Editor in the Creative SDK (recommended). To login and to access builds of the SDK you will now have to go to creativesdk.com.
We’re excited to continue building the Creative SDK and bringing you the best in creative tech. Please let us know if you have any questions by going to help.creativesdk.com.

**Comment by kaansoral** [comment]
I've created this issue, maybe something can come out of it: https://github.com/phonegap/adobe-creative-sdk-foundation/issues/6
